pkg/scheduler/scheduler.go

```diff
@@ -652,6 +651,7 @@ func (sched *Scheduler) scheduleOne() {
 			err := sched.bindVolumes(assumedPod)
 			if err != nil {
 				klog.Errorf("error binding volumes: %v", err)
+				sched.recordSchedulingFailure(assumedPodInfo, err, "VolumeBindingFailed", err.Error())
```

Comment:
the klog.Errorf on the previous line is redundant, we already log info on an error inside ```bindVolumes```, perhaps delete this and change that to klog.Error?

Reply:
Removed err from log. Added pod ns/name.

Reply:
why do we need to log two lines? lets just do it once here it is easier to grep if it is a single line.

Reply:
ditto to the one above as well.

Reply:
Sorry I didn't realize that the error reason is already provided to recordSchedulingFailure. Removed redundant log lines. And I found an existing redundant log which is removed as well!